### ISSUE TITLE
missing Group Policy to turn off passive mode, need Defender to be active enterprise wide

### ISSUE THREAD

**Issue by rykerabel** [MicrosoftDocs/microsoft-365-docs#7812]
Please confirm how to remove ForceDefenderPassiveMode using group policy.
We need Defender active and it is stuck in passive mode.
This is a huge security vulnerability.

Thank you.

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 881d2840-25a7-0f80-c62c-9240680314e2
* Version Independent ID: 881d2840-25a7-0f80-c62c-9240680314e2
* Content: [Troubleshooting issues when switching to Microsoft Defender for Endpoint](https://docs.microsoft.com/en-us/microsoft-365/security/defender-endpoint/switch-to-mde-troubleshooting?view=o365-worldwide#microsoft-defender-antivirus-is-getting-uninstalled-on-windows-server)
* Content Source: [microsoft-365/security/defender-endpoint/switch-to-mde-troubleshooting.md](https://github.com/MicrosoftDocs/microsoft-365-docs/blob/public/microsoft-365/security/defender-endpoint/switch-to-mde-troubleshooting.md)
* Product: **m365-security**
* Technology: **mde**
* GitHub Login: @denisebmsft
* Microsoft Alias: **deniseb**

**Comment by yogkumgit** [comment]
https://github.com/MicrosoftDocs/windows-itpro-docs/issues/7929
@denisebmsft Please help us. Thanks

**Comment by denisebmsft** [comment]
Hello @rykerabel. Sorry to hear that you are having issues with Microsoft Defender Antivirus (MDAV). 

The best way to set Microsoft Defender Antivirus to active mode manually is to set or edit the ForceDefenderPassiveMode registry key's value to 0.

1. On your Windows device, open Registry Editor as an administrator.
2. Go to Computer\HKEY_LOCAL_MACHINE\SOFTWARE\Policies\Microsoft\Windows Advanced Threat Protection.
3. Set or define a REG_DWORD entry called ForceDefenderPassiveMode, and set its value to 0.
4. Reboot the device.

If that doesn't work, your next step is to contact support (https://docs.microsoft.com/en-us/microsoft-365/admin/get-help-support).

I hope this helps!

Thank you so much for letting us know about this issue! I've updated the article with the information I've posted here and you'll see the updates live later today.
